### ISSUE TITLE
TODO: send Battery voltage

### ISSUE THREAD

**Issue by dontsovcmc** [x821938/UltraLowPower_WeatherStation#2]
I see example with Attiny85:
https://github.com/guibom/ESPTempLogger/blob/master/ESPTempLogger.ino#L413

**Comment by zoomx** [comment]
That code works if you calibrate the long value written in line 440
`  result = 1074835L / result;`
because the 1.1V reference is not very good.

**Comment by dontsovcmc** [comment]
Yes, you should check values and correct Constant. Or you can measure voltage by multimeter, attiny85 and add shift constant to server.
